distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/RemovePluginCommandTests.java

```diff
@@ -59,9 +59,13 @@ public void setUp() throws Exception {
     }
 
     static MockTerminal removePlugin(String name, Path home) throws Exception {
```

Comment:
I'd prefer to add the argument and pass `false` all the places that call this unless there are far more callers that I can see.